lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -1579,6 +1635,43 @@ instance LiftHandler ErrMetricsInconsistency where
                 , " but the node doesn't know about this stake pool!"
                 ]
 
+instance LiftHandler ErrCreateCert where
+    handler = \case
+        ErrCreateCertNoSuchPool poolId ->
+            apiError err404 PoolAlreadyJoined $ mconcat
+                [ "I couldn't find a stake pool with the given id: "
+                , toText poolId
+                ]
+        ErrCreateCertPoolAlreadyJoined poolId ->
+            apiError err404 PoolAlreadyJoined $ mconcat
```

Comment:
**[change required]**
This should rather be `err403`

**Why**
`404` is used for missing pages or resources. Here, the case is slightly different: the pool exists, the request is syntactically valid, but we refuse to proceed. That's the semantic given to `403 Forbidden`

Reply:
corrected